addons/survey/static/src/scss/survey_survey_views.scss

```diff
@@ -57,7 +61,7 @@
     // Ungrouped specific
     &.o_kanban_ungrouped {
         // Set a minimal height otherwise display may have different card sized
-        .o_survey_kanban_card_ungrouped {
+        .o_survey_kanban_card > div:first-child {
```

Comment:
Would rather have a specific anchor than use "div:first-child", a bit more robust. 
(Same for the one below).